Add Blog

dnlggr opened the pull request:
Adds a blog to the website.

~The code is ready for review, but the article is a draft still so don't merge this until we have an article to publish.~

~I have the shareholder letter ready but used the "Anatomy of a Bitcoin Transaction" draft in this PR to illustrate how images will work (since the shareholder letter doesn't have any).~

~Before we merge this PR, we can swap "Anatomy of a Bitcoin Transaction" with the Shareholder Letter if we want.~

dergigi reviewed (commented):
Love it! Looks great 😍 

Let's use the shareholder letter as the first blog entry. I think it would be great to have it as a non-PDF format. We could also link to the [Bitcoin Audible version](https://pod.link/1359544516/episode/8734b563dc083dbe289225733f8c8105) of the letter, now that I think of it 🤔

dnlggr commented on src/components/PostImage.js:
```diff
@@ -0,0 +1,32 @@
+  object-fit: contain;
+`
+
+const PostImage = ({ postId, name, caption }) => {
```
Made it optional! 👍

dnlggr commented on posts/2022-01-11-anatomy-bitcoin-transaction.md:
```diff
@@ -0,0 +1,79 @@
+---
+title: 'Anatomy of a Bitcoin Transaction'
+summary: 'When it comes to Bitcoin, misconceptions abound. This is not surprising. Bitcoin is a new thing, and, consequently, all metaphors and comparisons fall short, in one way or another.'
+date: '2022-01-11'
```
Sure good idea! 😄

dnlggr commented:
Alright should be ready for review.

- Made image captions optional
- Added block time
- Added the shareholder letter as first post and removed the "Anatomy of a ..." draft
- Added an example blog post (that won't be deployed but can be used as boilerplate for new posts)
- Fixed some UI issues

dergigi commented on netlify.toml:
```diff
@@ -1,5 +1,5 @@
 [build]
-  command = "npm run build && npm run export"
+  command = "rm -rf public/assets/blog/example && rm posts/example.md && npm run build && npm run export"
```
👍

dergigi reviewed (approved):
Wonderful! The example is great, thank you for that. 🤗 

Everything looks good, and I've just read the whole thing again using a text-to-speech plugin which worked perfectly. Let's put this baby live, we can always improve things as we go along. Onwards! 🚀
